code/noRAM/app/src/main/java/com/example/noram/EventListFragmentTemplate.java

```diff
@@ -1,6 +1,9 @@
 /*
 This file is used to provide a template for displaying list of events, that other fragments can use.
 Outstanding Issues:
+- Searchbar shows static searches (the list is not updated in real-time)
+- After a search, to get back a real-time list of all events, you need to manually click on the
+"ALL EVENTS" button (leaving the empty search bar will still keep the searchList visible)

```

Comment:
IMO, we should try to improve this at least a bit. One nice thing we could do is put an onResume() that puts us back to the default list whenever we resume. You could also maybe make it so that we go back to displaying the full list if text is changed and becomes empty? That might be feasible

Reply:
In a later commit, I actually modified the code so that it hid the search list if the search is empty (my mistake for forgetting to remove the header code).
I'll add a onResume() method who hides the searchList by default.